src/main/java/org/zone/region/flag/entity/player/move/message/greetings/GreetingsFlag.java

```diff
@@ -0,0 +1,99 @@
+package org.zone.region.flag.entity.player.move.message.greetings;
+
+import net.kyori.adventure.text.Component;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.zone.region.flag.Flag;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.flag.entity.player.move.message.display.MessageDisplay;
+import org.zone.region.flag.entity.player.move.message.display.MessageDisplayTypes;
+
+/**
+ * Flag for greetings
+ */
+public class GreetingsFlag implements Flag {
+
+    private @NotNull MessageDisplay messageDisplay = MessageDisplayTypes.CHAT.createCopyOfDefault();
+    private @NotNull Component text;
+    @Deprecated(forRemoval = true)
+    private @Nullable Component legacyText;
```

Comment:
Why is there legacyText in a new class? Also for backwards compatibility you cannot move a class

Reply:
where did i move bruh

Reply:
```shell
src/main/java/org/zone/region/flag/entity/player/move/message/greetings/GreetingsFlag.java
src/main/java/org/zone/region/flag/entity/player/move/greetings/GreetingsFlag.java
```

The latter is the original, with the former being your moved file. 
I said ``new class`` as it effectivly is as the original was deleted with this new class taking its place, sure the code is based upon the removed file, but moving a file is the same as coping the old file, deleting it then creating the new file and pasting in the old file.

Reply:
this has been fixed tho